packages/drafts-realm/sale-hub.gts

```diff
@@ -0,0 +1,1070 @@
+        display: flex;
+        gap: var(--boxel-sp-xxs);
+        overflow-x: auto;
+      }
+      .activity-lists {
+        margin-top: 1rem;
+      }
+      .activity-lists > * + * {
+        margin-top: 1rem;
+      }
+      .activity-list {
+        padding: var(--boxel-sp-xs);
+      }
+      .sub-heading {
+        display: flex;
+        align-items: start;
+        justify-content: space-between;
+
+        font-weight: bold;
+        padding: var(--boxel-sp-xxs);
+        background-color: var(--boxel-200);
+      }
+      .relationship-tab-panel {
+        padding: var(--boxel-sp-xs);
+        border-radius: var(--boxel-border-radius);
+        background-color: var(--boxel-100);
+      }
+      .button {
+        display: flex;
+        flex: 0 0 auto;
+        width: auto;
+        min-width: 120px;
+        flex-wrap: wrap;
+        align-items: center;
+        justify-content: center;
+        gap: var(--boxel-sp-xs);
+        padding: var(--boxel-sp-xxs);
+        font-size: var(--boxel-font-size-sm);
+        cursor: pointer;
+        font-weight: 600;
+        position: relative;
+        background-color: var(--boxel-100);
+        border: 2px solid var(--boxel-200);
+      }
+      .button:hover {
+        background-color: var(--boxel-200);
+      }
+      .button-save {
+        width: 100%;
+        background-color: var(--boxel-dark-teal);
+        border: 2px solid var(--boxel-200);
+        color: white;
+        padding: var(--boxel-sp-xxs);
+        font-size: var(--boxel-font-size-sm);
+        font-weight: 600;
+        cursor: pointer;
+      }
+      .dialog-box {
+        max-height: 80vh;
+        overflow: overlay;
+      }
+      .line-through {
+        text-decoration: line-through;
+      }
+      .checkbox-card {
+        display: flex;
+        align-items: start;
+        justify-content: space-between;
+        gap: var(--boxel-sp-xs);
+      }
+      .dueDate {
+        color: var(--boxel-red);
+      }
+      .formInputGroup > * + * {
+        margin-top: 1.5rem;
+      }
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+      .footer {
+        display: flex;
+        align-items: center;
+        justify-content: end;
+        gap: var(--boxel-sp-xs);
+      }
+    </style>
+  </template>
+}
+
+export class SaleHub extends CardDef {
+  static displayName = 'sale hub';
+
+  @field targetPage = contains(StringField, {
+    description: `Show which page is clicked`,
+  });
+  @field leadForm = contains(LeadFormField, {
+    description: `Lead form`,
+  });
+  @field accountForm = contains(CrmAccountField, {
+    description: `crm account`,
+  });
+  @field contactForm = contains(ContactFormField, {
+    description: `Contact Form`,
+  });
+  @field opportunityForm = contains(OpportunityFormField, {
+    description: `Opportunity Form`,
+  });
+  @field scheduledTask = containsMany(ScheduledTask, {
+    description: `Upcoming & overdue tasks`,
+  });
```

Comment:
Ok. I think this one single comment is going to require a lot of changes. Can you change the contains objects into linksTo? These cards already exist -- Lead, Account, Contact, Opportunity, Task